src/app/index.js

```diff
@@ -1,117 +1,107 @@
-const gui = require('nw.gui');
 const gkm = require('gkm');
-const pjson = require('../package.json');
 const AudioStreamMeter = require('audio-stream-meter');
-const character = document.getElementById("character");
-var exportable = require("./twitchAuth.js");
 
-//Tesing twitch auth file
-exportable.doSomething();
+const exportable = require('../app/twitchAuth.js');
```

Comment:
`./` doesn't work because it isn't relative to this file (`__dirname`), it's relative to the current working directory (`process.cwd()`). So it needs to be `../app/`